good-enough-practices-for-scientific-computing.tex

```diff
@@ -536,7 +544,7 @@ \subsection*{Acknowledgments}
   to.
 
 \item
-  \practice{3b}{Create a shared public ``to-do'' list}.  This can be a
+  \practice{3b}{Create a shared ``to-do'' list}.  This can be a
```

Comment:
This was in response to the reviewer who thought this whole section only applied to open source projects. I think these practices also apply when working privately (i.e. if new collaborators are only people joining the lab).